src/plugins/invitationmanager/demo/im.json

```diff
@@ -0,0 +1,179 @@
+﻿{
+  "settings":
+  {
+    "duration-delay": 15,
+    "page-probability": 0.7,
+    "site-probability": 0.3,
+    "blacklist": [{"url" : ""}],
+    "whitelist": [{ "url" : "^localhost\/"}]
```

Comment:
Can you please add the GCWeb website hosted by GitHub in the whitelist for the purpose of demoing?

**wet-boew.github.io**